README.md

```diff
@@ -66,6 +59,14 @@ Content for Spartacus pages is fetched from the SAP Commerce Cloud CMS (Content
 
 The documentation for customizing and extending Spartacus is still under development and is being released as it becomes available.
 
+# API Documentation
+
+The Spartacus API documentation is hosted here: [https://sap.github.io/spartacus/](https://sap.github.io/spartacus/)
+
+The API documentation is versioned and is included in the **Assets** folder of each and every Spartacus library. You can download the API documentation for a particular version by clicking on `docs.tar.gz` or `docs.zip` in the **Assets** folder of any Spartacus library from that particular release.
+
+For example, to download the API documentation for the 2.0.0 release, you could access the **Assets** section of the `@spartacus/storefront@2.0.0` library [here](https://github.com/SAP/spartacus/releases/download/storefront-2.0.0/docs.zip).
+
 # Limitations
 
```

Comment:
Maybe rewrite this to say that Spartacus releases 1.x and 2.x work with 1905 and 2005 only. Certain features are only available with 2005 because of OCC API support, such as cancellations and returns.

Reply:
As discussed, this comment has been added as a note in the API Documentation section 🙂